__DEFINES/reagents.dm

```diff
@@ -388,6 +388,8 @@
 #define APETRINE		"apetrine"
 #define SODIUMSILICATE	"sodiumsilicate"
 #define COLORFUL_REAGENT "colorful_reagent"
+#define AMINOMICIN		"aminomicin"
+#define SYNTHMOUSE		"synthmouse"
```

Comment:
You defined this but you don't use it. I don't think you're supposed to define recipe IDs anyway

Reply:
I do use it, for the recipe ID. Most everything else seems to be using a recipe ID define so I just made one too